src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -144,7 +157,8 @@ export class BucketDetailComponent implements OnInit {
           let str = res._body;
           let x2js = new X2JS();
           let jsonObj = x2js.xml_str2json(str);
-          let alldir = jsonObj.ListBucketResult ? jsonObj.ListBucketResult :[] ;
+          let alldir = jsonObj.ListBucketResult && jsonObj.ListBucketResult.Contents ? jsonObj.ListBucketResult.Contents :[] ;
```

Comment:
This file is updated in #232 . 
We will have to merge that first and then rebase this PR.
@kumarashit @wisererik @rhsakarpos  Please review #232

Reply:
I just confirmed with the backstage staff that the value should be placed in 'content', consistent with AWS

Reply:
Yes. This is already addressed in #232 Please check.

Reply:
I have made the change, plz check it, thank you.